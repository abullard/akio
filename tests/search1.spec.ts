import { describe, it, expect } from "vitest";
import { spawnWrapper } from "./test-utils";

describe('search1.spec.ts', () => {
    it('should run search for start npm scripts, skip input', async () => {
        const skipInput = '-i';
        const cmd = "pnpm";
        const args = ['akio', 'start', skipInput];

        const { stdout } = await spawnWrapper(cmd, args);

        expect(stdout).toMatchSnapshot();
    });
});
