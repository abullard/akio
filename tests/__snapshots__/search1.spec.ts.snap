// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`search1.spec.ts > should run search for test npm scripts, skip input 1`] = `
"
> node ./dist/index.cjs build -i

[33mpnpm akio[0m
	-----
ğŸ“¦ [34m@abullard/akio[0m
1. [35mbuild:real[0m â€” This command builds Akio

ğŸ“¦ [34mapi[0m

ğŸ“¦ [34mui[0m
"
`;
